<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CnewAccountDlg</class>
 <widget class="QDialog" name="CnewAccountDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>148</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="m_okButton">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="m_cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="m_remitBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>631</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Remittance Setup</string>
   </property>
   <widget class="QLabel" name="m_localShareLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Local Share Perentage</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_hqShareLbl">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HQ Share Percentage</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_localShare">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_hqShare">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_remittanceName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_remitNmeLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Remittance Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="m_useHqReceipt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>118</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Hq Receipt</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_paymentAccuntLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>105</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment Account</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_paymentAccountCombo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>401</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_bankAccountNoLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>104</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bank Account No</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_bankAccountNo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="m_deptLbl">
      <property name="text">
       <string>Department</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_accountNameLbl">
      <property name="text">
       <string>Account Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_accountType">
      <property name="text">
       <string>Account Type</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>401</width>
     <height>99</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="m_deptCombo"/>
    </item>
    <item>
     <widget class="QLineEdit" name="m_accountName"/>
    </item>
    <item>
     <widget class="QComboBox" name="m_accountTypeCombo"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_deptCombo</tabstop>
  <tabstop>m_accountName</tabstop>
  <tabstop>m_accountTypeCombo</tabstop>
  <tabstop>m_useHqReceipt</tabstop>
  <tabstop>m_paymentAccountCombo</tabstop>
  <tabstop>m_bankAccountNo</tabstop>
  <tabstop>m_cancelButton</tabstop>
  <tabstop>m_okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
